crates/tokio_helper/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+use once_cell::sync::Lazy;
+use tokio::runtime::Runtime;
+
+pub static TOKIO: Lazy<Runtime> = Lazy::new(new_runtime);
+
+/// Instantiate a new runtime.
+pub fn new_runtime() -> Runtime {
+    // we want to use multiple threads
+    tokio::runtime::Builder::new_multi_thread()
+        // we use both IO and Time tokio utilities
+        .enable_all()
+        // we want to use thread count matching cpu count
+        // (sometimes tokio by default only uses half cpu core threads)
+        .worker_threads(num_cpus::get())
```

Comment:
I vote we delete this - we can allow the tokio default now until/unless we prove it's actually better to increase this. They have done more testing than us.